packages/gitmoji-changelog-cli/src/cli.js

```diff
@@ -131,4 +137,62 @@ function logMetaData(changelog) {
   }
 }
 
+async function handleUnexpectedErrors(options, projectInfo, repository, e) {
+  const envInfo = await envinfo.run(
+    {
+      System: ['OS'],
+      Binaries: ['Node', 'Yarn', 'npm'],
+    },
+    { markdown: true }
+  )
+
+  const copyToClipboard =
+    // Clipboard is not accessible when on a linux tty
+    process.platform === 'linux' && !process.env.DISPLAY
+      ? false
+      : true
+
+
+  const issueBody = `${envInfo}
+
+## Stack trace
+
+\`\`\`
+${e.stack}
+\`\`\`
+
+## CLI options
+
+${table([
+  ['Option', 'Value'],
+  ...Object.entries(options),
+])}
+
+## Project info
+
+${table([
+  ['Key', 'Value'],
+  ...Object.entries(projectInfo),
+])}
+
+## Repository info
+
+${table([
+  ['Key', 'Value'],
+  ...Object.entries(repository),
+])}`
+
+  const url = newGithubIssueUrl({
+    user: 'frinyvonnick',
+    repo: 'gitmoji-changelog',
+    body: 'Thank you for reporting your issue :+1:\n\nThe bug report is in your clipboard, please paste it here.',
```

Comment:
Maybe using a Github template could be great here, in order to have a more detailed message without having a big url.
See https://github.com/sindresorhus/new-github-issue-url#template
You can also automatically add a label ;)

Reply:
I opened an issue on [issue-reporter](https://github.com/frinyvonnick/issue-reporter/issues/3)